themes/shezaan/shezaan.theme.bash

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+SCM_THEME_PROMPT_DIRTY=" ${red}✗"
+SCM_THEME_PROMPT_CLEAN=" ${bold_green}✓"
+SCM_THEME_PROMPT_PREFIX=" |"
+SCM_THEME_PROMPT_SUFFIX="${green}|"
+
+GIT_THEME_PROMPT_DIRTY=" ${red}✗"
+GIT_THEME_PROMPT_CLEAN=" ${bold_green}✓"
+GIT_THEME_PROMPT_PREFIX=" ${green}|"
+GIT_THEME_PROMPT_SUFFIX="${green}|"
+
+RVM_THEME_PROMPT_PREFIX="|"
+RVM_THEME_PROMPT_SUFFIX="|"
+
+myip=`$BASH_IT/lib/get_ip.py`
```

Comment:
Based on your responses to the inclusion of a simple python script above. We can discuss your preference on how to want to handle this part.
Whether or not this commit is merged, this is a great project. Keep up the great work and best of luck.